core/domain/role_services.py

```diff
@@ -108,6 +108,17 @@
 }
 
 
+def get_human_readable_role(role):
```

Comment:
If you do this, I think it'll be good to also add a comment where the role constants are defined about the assumed format (i.e. that they are the human readable roles with underscores instead of spaces)